docs/tutorials/configured-instances.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 10
+title: Configuring Class Instances
+description: Learn how to make workflows, transactions, and communicators reusable and configurable
+---
+
+In this guide, you'll learn how to make your DBOS functions configurable using instances.
+
+## Concepts
+Basic DBOS transactions, communicators, and workflows are just functions - they accept input parameters, apply those parameters to the database, or use them to place calls to external services.
+
+However, it is sometimes desirable to have configuration information available to DBOS functions.  Using function parameters for items such as access URLs, API keys, port numbers, and so on is a bad idea, so these are generally stored as configuration data in `dbos-config.yaml` or otherwise accessed from context.
+
+If a function needs more than one configuration, such global settings are not adequate.  For example, an email-sending function may send email with one set of addresses and credentials for promotional materials to prospects, or another set of credential for replies to support inquiries from existing customers.
+
+### Instances
+Configured class instances are the DBOS Transact mechanism for creating multiple function configurations.  Rather than having `static` class member functions, configured instances are TypeScript object instances with non-static member functions that can access configuration information through `this`.
```

Comment:
They're TypeScript classes, no?

Reply:
What is the proposed wording?

Reply:
"Configured instances are TypeScript classes with non-static member functions..."